pubspec.yaml

```diff
@@ -28,13 +28,15 @@ environment:
 # the latest version available on pub.dev. To see which dependencies have newer
 # versions available, run `flutter pub outdated`.
 dependencies:
+  provider: ^6.0.0
```

Comment:
why do you have provider added? are you using it? if not remove it please

Reply:
fixed